### DISCUSSION TITLE
Spanish translation of Litematica 🇪🇸

### DISCUSSION THREAD

**Discussion by pilahito** [maruohon/litematica#923] [General]
[es_es.json](https://github.com/user-attachments/files/18135464/es_es.json)

*Spelling corrections
*What arrangement where the texts were sticking out of their gap or the gui
*Some accents and some text that were not very meaningful in the translation were added.

**Comment by sakura-ryoko** [comment]
Added to 1.21.4 branch and then 1.21 / 1.21.3.

https://github.com/sakura-ryoko/litematica/tree/ae67326d9988db39632bf54eacfb96afe93e6013/src/main/resources/assets/litematica/lang

There are more English entries that have not been translated yet.
